### DISCUSSION TITLE
A propriedade "DataSetAdapter" não tem mais?

### DISCUSSION THREAD

**Discussion by MrWolfX360** [viniciussanchez/RESTRequest4Delphi#158] [Q&A] [answered]
Olá bom dia...

Eu fiz a atualização do RESTRequest4Delphi, e nos meus projetos antigos, ele acusa erro, na propriedade DataSetAdapter... Não há mais suporte a este recurso? 
Ou há outra forma de fazer?

Agradeço pela atenção... Abraços

**Comment by viniciussanchez** [comment, marked as answer]
Bom dia... deixamos o projeto desacoplado com a possibilidade de quem usa criar seus próprios providers, sem ficar amarrado ao DataSet-Serialize. Agora pode ter seu próprio parser... mas para usar como antes: 

![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/16382981/5dfc9d75-a5e0-4d37-bd27-556439e9ef7a)
![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/16382981/0403a86a-6579-4737-bd71-4f5773d109e0)

**Reply by MrWolfX360** [reply]
Entendi... Maravilha... Concordo, fica mais clean... 

Agradeço pela atenção...

Muito obrigado... Abraços
